abc.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project</title>
+    <link rel="stylesheet" href="abc.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    
+</head>
+<body>
+<header><div class='nav-border-container'>
+    <div class="nav-logos"><img src="images/facebook-logo.png" alt=""><img src="images/twitter.png" alt=""><h2 class="nav-head">English</h2><h3 class='nav-head'>My Page</h3><h4 class='nav-head'>Logout</h4></div>
```

Comment:
- [ ] Kindly add a description of the image in all of the `alt` attributes.

Reply:
fixed